components/IndoorDirections/TypesOfDirections/DifferentBuildingDirections.js

```diff
@@ -1,25 +1,31 @@
 import React from 'react';
 import { Line, G } from 'react-native-svg';
 import { ClassGraph } from '../../../constants/ClassGraph';
+import { Class9Graph } from '../../../constants/ClassGraph9';
 import { dijkstra, getFloorNumber, ConvertToHall8Floor } from '../Dijkstra/DijkstraAlgorithm';
+import { Hall9Coordinates } from '../../../constants/Hall9Coordinates';
```

Comment:
So for every floor that we add, we will have to import it. We can't have all the floors in the same imports ish? I think it's going to be hard to extend.

Reply:
We could, but for the purpose of this project, we separate it

Reply:
He has a good point, it would be hard to extend if we were to have a lot of floor maps for multiple buildings

Reply:
Indeed, but, what we could do is store all the data files into one. So, there would be one ClassGraph.js & one SVGCoordinates.js

Reply:
Exactly!

Reply:
Yes, but it would take me hours. Can we keep this idea if JMSB is required in sprint 4?

Reply:
Okay I give you permission